roles/dont_use_groups/playbook.yml

```
@@ -0,0 +1,26 @@
+---
+- name: show how to loop through a set of groups
+  hosts: cluster_group_?
+  gather_facts: false
+  become: false
+
+  tasks:
+  - name: the loop happens for each host, might be too much
```

We have a rule to generally indent list elements (see https://github.com/redhat-cop/automation-good-practices/blob/mophahr--remove_rule_for_j2/coding_style/README.adoc#yaml-and-jinja2-syntax):

```suggestion
    - name: the loop happens for each host, might be too much
```
…and so on

---

You're right. Done on the command line.